backend/service/martini/email.go

```diff
@@ -0,0 +1,111 @@
+package martini
+
+import (
+	"bytes"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/ekspand/trusty/api"
+	v1 "github.com/ekspand/trusty/api/v1"
+	"github.com/ekspand/trusty/internal/db"
+	"github.com/ekspand/trusty/pkg/email"
+	"github.com/go-phorce/dolly/rest"
+	"github.com/go-phorce/dolly/xhttp/httperror"
+	"github.com/go-phorce/dolly/xhttp/identity"
+	"github.com/go-phorce/dolly/xhttp/marshal"
+)
+
+const (
+	userEmailSubject    = "You have successfully created the validation request."
+	contactEmailSubject = "Do you approve issuance of certificates for your organization?"
+)
+const defaultHTMLTemplateForUserEmail = `
+<h1>
+    You will need to provide the contact this number fo complete the validation.
+    <h2>{{.Number}}</h2>
+</h1>
+`
+
+const defaultHTMLTemplateForContactEmail = `
+<h1>
+    {{.Email}} has requested permission to acquire certificates for your organization.
+    <h2>{{.OrgDetails}}</h2>
+    If this request was authorized, enter the number that {{.Email}} provided you below and click "Approve".
+</h1>
+`
+
+// SendEmailHandler handles v1.PathForMartiniSendEmail
+func (s *Service) SendEmailHandler() rest.Handle {
```

Comment:
it should not be public

what I meant before, `martini` should have a folder with templates,
and will call `email` package.
but marini should not expose end-point